### PR TITLE
Nr 1308 field newsroom item id as guid

### PR DESCRIPTION
**Pull request by planctus**
This pull requests includes a different version of the feeds module (the current stable one). That means that it needs a rebuild of the platform to be tested. (or a manual update of the module, eventually)

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info [hunk 1]
```diff
@@ -4,20 +4,15 @@ core = 7.x
 package = NextEuropa
 version = 7.x-2.0
 project = nexteuropa_newsroom
-dependencies[] = entity_translation
```
Comment (LOBsTerr) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info]: Why did you delete entity_translation and title dependency?

File: profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module [hunk 2]
```diff
@@ -1979,15 +2010,7 @@ function nexteuropa_newsroom_feeds_after_save(FeedsSource $source, $entity, $ite
...
-            // Delete translation for the current language.
-            $handler = entity_translation_get_handler('node', $entity);
-            $handler->removeTranslation($language->language);
-            field_attach_presave('node', $entity);
-            field_attach_update('node', $entity);
-
-            // It is necessary to clean the cache
-            // to remove the translation completely.
-            cache_clear_all($entity_id, 'cache_entity_node');
+            _nexteuropa_newsroom_delete_translations($entity, $entity_id);
```
Comment (LOBsTerr) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module]: I think it is wrong. We remove translation, which doesn't have any values, but _nexteuropa_newsroom_delete_translations goes through all languages and remove all translations

Comment (planctus) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module]: because they are enabled by the platform, no need to use them as a dependency in our feature then.

File: profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info (hunk 1 above)
Comment (planctus) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info]: because they are enabled by the platform, no need to use them as a dependency in our feature then.

File: profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module (hunk 2 above)
Comment (planctus) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module]: @LOBsTerr, if i'm correct i didn't change anything in the existing implementation, i've just created an helper function in order for it to be usable elsewhere. I do agree, in general, that there would be a better approach, maybe, for deleting translations, but that's what i've found in place and i reused it

Comment (planctus) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.module]: Ah, no..now i see, is actually doing something crazy, i'll fix this

File: profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info (hunk 1 above)
Comment (LOBsTerr) [inline profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info]: I see it is available in the platform, but I don't see it is enabled by default. So, I think we should keep it, in order have it enabled.
